Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v2/FeedOpenCTI_v2_description.md

```diff
@@ -0,0 +1,23 @@
+## Configure OpenCTI Feed v2 on Cortex XSOAR
```

Comment:
Is this a copy of the documentation? Why do we need the same info as in the `?`?

Reply:
changed description